src/rate.js

```diff
@@ -1,24 +1,49 @@
-import { sortBy, identity } from 'ramda'
+import { sortBy, identity, range } from 'ramda'
 import unwind from 'sort-unwind'
 import models from './models'
 
 const rate = (teams, options = {}) => {
   const model = models[options.model || 'plackettLuce']
+  let processedTeams = teams
 
-  // if no rank or score provided, use natural ordering
-  if (options.rank === undefined && options.score === undefined) {
-    return model(teams, options)
+  // if tau is provided, use additive dynamics factor to prevent sigma from dropping too low.
+  // using this will ensure the rating will stay more pliable after many games
+  if (options.tau) {
+    const tauSquared = options.tau * options.tau
+    processedTeams = teams.map((team) =>
+      team.map((p) => ({
+        mu: p.mu,
+        sigma: Math.sqrt(p.sigma * p.sigma + tauSquared),
+      }))
+    )
   }
 
   // if rank provided, use it, otherwise transition scores and use that
-  const rank = options.rank ?? options.score?.map((points) => -points)
+  const rank =
+    options.rank ??
+    options.score?.map((points) => -points) ??
+    range(1, teams.length + 1)
```

Comment:
This was only necessary because I removed the if statement that handled undefined values before. The reason I removed the if statement was to process the `peventSigmaIncrease` flag every time.